builtin/logical/consul/path_token.go

```diff
@@ -103,13 +106,41 @@ func (b *backend) pathTokenRead(ctx context.Context, req *logical.Request, d *fr
 		})
 	}
 
+	aclServiceIdentities := []*api.ACLServiceIdentity{}
+	parsedServiceIdentities, err := parseSemicolonStringSlice(roleConfigData.ServiceIdentities)
+	if err != nil {
+		return nil, err
+	}
+	for _, serviceIdentity := range parsedServiceIdentities {
+		entry := &api.ACLServiceIdentity{}
+		components := strings.SplitN(serviceIdentity, ":", 2)
```

Comment:
Why is `strings.SplitN` 2 here for `n`?

Reply:
It made sense to keep this at first, because it would ensure that the string is only ever split into 2 parts on the first colon. After thinking about it more, I believe there there should only ever be one colon per serviceIdentity in this section of code when used correctly, so Split vs SplitN should have no difference.

Given that, I'd rather change it to regular Split for code clarity.